### PR TITLE
Code fixes for return, break statements

### PR DESCRIPTION
**Pull request by sapkotamadan**
Hi Please review the updates. The comment is added in each commits.

### CODE REVIEW

File: system/I18n/Time.php [hunk 1]
```diff
@@ -278,7 +278,7 @@ public static function create(int $year = null, int $month = null, int $day = nu
 	 *
 	 * @return \CodeIgniter\I18n\Time
 	 */
-	public static function createFromFormat($format, $datetime, $timeZone = null)
+	public static function createFromFormat($format, $datetime, DateTimeZone $timeZone = null)
```
Comment (lonnieezell) [inline system/I18n/Time.php]: Are you actually receiving an error anywhere from this? I don't in php 7.0.22, and we want the functionality to be able to simply pass a string here.

**Comment by jim-parry** [comment]
Being picky, but we expect commits to be GPG-signed :)

**Comment by jim-parry** [comment]
The return statements eliminated might be needed for testing. I know that the one in Console is, and cannot be simply removed.

**Comment by lonnieezell** [comment]
No response from OP for 4 months. Don't get any errors locally or on Travis. Closing.

**Pull request closed without merging**